src/components/Home/Hero/HeroTitleSection.tsx

```diff
@@ -0,0 +1,55 @@
+import cn from 'classnames'
+import React from 'react'
+
+import * as styles from './styles.module.css'
+
+const HeroTitleSection = ({ className }: { className?: string }) => {
+  return (
+    <div className={className}>
+      <div
+        className={cn(
+          'text-center ',
+          'px-6 py-10 md:py-24',
+          'flex flex-col gap-6 md:flex-row md:gap-0',
+          'max-w-screen-lg'
+        )}
+      >
+        <div
+          className={cn(
+            'text-indigo',
+            'flex-1',
+            'md:border-r md:border-solid md:border-r-light',
+            'md:text-left',
+            'lg:mr-16'
+          )}
+        >
+          <h1 className={cn('leading-10 font-extralight', styles.title)}>
```

Comment:
These lines look a little squashed. Can you increase the line-height of the `h1` to 1.5 and add a 16px margin between the `h1` and the `p` which follows?